R/individuals.R

```diff
@@ -101,79 +101,62 @@ create_states <- function(parameters) {
 #' @param parameters, model parameters created by `get_parameters`
 #' @importFrom stats rexp rnorm
 create_variables <- function(parameters) {
-  initial_age <- trunc(
-    rexp(
-         parameters$human_population,
-         rate=1/parameters$average_age
-    )
-  )
+  size <- parameters$human_population
+
+  initial_age <- trunc(rexp(size, rate=1/parameters$average_age))
```

Comment:
I think they're equivalent for positive floating point numbers, but I feel `floor` reflects the mathematics here a bit closer.